test/razorpay/test_payment.rb

```diff
@@ -298,6 +298,12 @@ def test_validate_vpa
       stub_post(%r{payments/validate/vpa$}, 'fake_validate_vpa',param_attr.to_json) 
       payment = Razorpay::Payment.validate_vpa param_attr.to_json 
       assert_equal param_attr[:vpa], payment.vpa
+    end
+    
+    def test_expendDetails
```

Comment:
can you also add test cases for other options like cards and offers and error scenarios

Reply:
Added for card